compaction/compaction.cc

```diff
@@ -1399,6 +1399,32 @@ class scrub_compaction final : public regular_compaction {
     std::string _scrub_finish_description;
     uint64_t _bucket_count = 0;
```

Comment:
I don't understand the conditions. Let's say the total amount of data is 1TB and partitions are 1GB. Why not use memtables? Provided we can close+open, which I think we can.